platformio_user.ini.sample

```diff
@@ -30,6 +30,9 @@
 ;build_type = debug
 ;monitor_port = COM3
 ;upload_port = COM3
+;build_flags =
```

Comment:
This change will require approval from Shelby. The Secrets file is an integral part of the build process and is understood by the people working with the system.

Reply:
Issue:
When cloning the depot, opening in PlatformIO, the build ***fails***.

Needs:
"people working with the system" need to include a "secrets.h" file
the depot should build w/o error

Alternate solution 1:
* Add a blank secrets.h to the depot containing the defaults, but also prevent any future modifications to the file
* To check: Can .gitignore can do this, even after the file is in the depot?
* To check: Other ways to "lock" a file?

Alternate solution 2:
* If a symbol is defined, then include the header file.
* By default, do NOT include the header file.
* "people working with the system" would define the symbol via platformio_user.ini (or the like)

As it is, having a clean clone of the depot not build is ... not a great experience.

Reply:
Confirmed.  If a secrets.h file with no real information is checked in (`git add -f secrets.h`), it will prevent the build break.

Where should the "secrets.h" file exist, if it's used?  (same directory as the .INO file?)

Reply:
Highlighted review request for this in main summary.